src/main/java/com/microsoft/graph/Util.java

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.graph;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+public final class Util {
+    private Util() {}
+
+    public static void closeQuietly(Closeable closeable) {
```

Comment:
Added this because the one from okhttp handles only checked exceptions and doesn't handle null objects.